keras_nlp/src/models/resnet/resnet_backbone.py

```diff
@@ -155,21 +169,21 @@ def __init__(
         # The padding between torch and tensorflow/jax differs when `strides>1`.
         # Therefore, we need to manually pad the tensor.
         x = layers.ZeroPadding2D(
-            3,
+            1 if use_vd_pooling else 3,
```

Comment:
Under the proposal above, this might be able to be written as `input_conv_kernel_sizes[0] // 2` or something like that?